docs/docs/cmd/spp/model/model-get.mdx

```diff
@@ -0,0 +1,306 @@
+  Notice"],"caseSensitive":false,"ignoreDigitIdentity":false,"ignoreLetterIdentity":false}]}},
+    "ID": 3,
+    "LastTrained": "2020-11-03T18:43:53Z",
+    "ListID": "bc696b17-378c-4347-b509-5d52cc6f3823",
+    "ModelSettings": null,
+    "ModelType": 2,
+    "Modified": "2024-06-27T02:45:32Z",
+    "ModifiedBy": "i:0#.w|sharepoint\\system",
+    "ObjectId": "01XHOXSCXFBUHLC2OA7FDJB52PXCWAAE3S",
+    "PublicationType": 0,
+    "Schemas": {"Extractors":{"Adressee":{"concepts":{"0a18ef8b-c0cd-435d-8a45-37738fef4f6f":{"name":"Adressee"}},"relationships":[]},"Discount Percentage":{"concepts":{"48c09840-d2da-4416-a02d-68113da24d18":{"name":"Discount Percentage"}},"relationships":[]},"Date of Change":{"concepts":{"f39b4e2a-e945-4a2e-99a0-561334037a66":{"name":"Date of Change"}},"relationships":[]},"Date Completed":{"concepts":{"3559972a-d7c5-4ccb-b8cb-e4b852da2399":{"name":"Date Completed"}},"relationships":[]},"Contributor":{"concepts":{"6af8ef33-4bf1-49ca-93d3-7a2358746747":{"name":"Contributor"}},"relationships":[]}}},
```

Comment:
For all properties that contain a JSON object, let's format them the usual way instead of putting everything on one line.